### ISSUE TITLE
Using a dropdown to select variables rather than drag and drop

### ISSUE THREAD

**Issue by rsenft1** [dreamRs/esquisse#233]
Love equisse! I'd like to be able to use it with datasets that have hundreds of columns though, and the drag and drop interface gets overpopulated. Is there a way to feed variables from a dropdown menu or other shiny UI element to equisse rather than using the drag and drop? Or to populate the drag and drop options by multiselecting a dropdown os other UI element? Thanks!

**Comment by pvictor** [comment]
It's a good idea, we could have a threshold at which we automatically switch to drop down menus if there are too many variables, with potentially an argument to force the choice between drag and drop and drop down menu.

**Comment by kb2010** [comment]
Another idea would be to have the list of variables on the left had side as a long column with a search box at the top of the column and the list if scrollable. would make it so easy for data sets with lots of columns. Added option could be to have option to categorise the fields into the types so its easy to go right to the Colum you want.

**Comment by dabsingh** [comment]
I agree with kb2010, it would be great if there was a way to get the variable selection on the left side using a scrollable  list with a searchbox. With the current UI is, it is very hard to find the variables to drag and drop when the data has a lot of variables. The other option is to make each X,Y,fill,color,size,group,facet a dropdown list (with search). Would appreciate if someone can incorporate these. Esquisse is an amazing package...hope it continues to evolve.
